src/liger_kernel/ops/utils.py

```diff
@@ -68,3 +68,39 @@ def compare_version(package: str, operator: Callable, target: str):
     torch.float16: tl.float16,
     torch.bfloat16: tl.bfloat16,
 }
+
+
+@triton.jit
+def element_mul_kernel(
```

Comment:
maybe we can delete the one in the original [ce](https://github.com/linkedin/Liger-Kernel/blob/main/src/liger_kernel/ops/cross_entropy.py#L163) and share this